### PR TITLE
Remove consent banner from simorgh

### LINKED ISSUE #2100: Remove Consent banner component from Simorgh
**Is your feature request related to a problem? Please describe.**
Following this issue: https://github.com/bbc/simorgh/issues/1934 ConsentBanner component has been moved to Psammead therefore it can be removed from the Simorgh repo.

**Describe the solution you'd like**
Remove the ConsentBanner component

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Regression on the component 

**Additional context**

### PR DESCRIPTION
**Pull request by thekp**
Resolves #2100

**Overall change:** 

_From this [PR](https://github.com/bbc/psammead/pull/667), the ConsentBanner component has been moved to Psammead, therefore this PR is to remove the component from Simorgh_

_**note:** the banner is conditionally rendered, If you can not see
      the banner above please clear your cookies or view in incognito mode._

**Code changes:**

- _Removed ConsentBanner Component._
- _Updated the package.json to have the Psammead ConsentBanner Component._
- _Updated import paths_
- _Add a storybook example of the consentBanner container_
- _Add some required scripts to the  ampDecorator so that the ConsentBanner works on storybook_

**Testing Instructions:**
Regression of the component should be done. The component can be seen by `npm run dev` and then checking (in-private browser or clear cookies):
- http://localhost:7080/news/articles/c6v11qzyv8po
- http://localhost:7080/news/articles/c6v11qzyv8po.amp

I have also added the container logic for the component to storybook, however there is a known bug with AMP components on story book and this issue is already raised: https://github.com/bbc/simorgh/issues/1698

--

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by thekp** [comment]
There is currently an problem with the storybook I have created for the ConsentBanner Container for AMP.

The bug is that when a user clicks the specific container on storybook, then click on a different one and return to the ConsentBanner container for AMP then it will not work. This is because it is trying to apply analytics twice, which in reality would not happen during normal user interactions

similar issue: https://github.com/bbc/simorgh/issues/1698

**Comment by thekp** [comment]
Example of bug
ezgif com-video-to-gif

File: .storybook/helpers/ampDecorator/index.jsx [hunk 1]
```diff
@@ -18,6 +18,21 @@ const AmpDecorator = storyFn => (
       <style amp-boilerplate="">{AMP_SCRIPT}</style>
       <noscript>{`<style amp-boilerplate="">${AMP_NO_SCRIPT}</style>`}</noscript>
       <script async src="https://cdn.ampproject.org/v0.js" />
+      <script
```
Comment (dr3) [inline .storybook/helpers/ampDecorator/index.jsx:21]: Can you write an issue to dedupe these between here and https://github.com/bbc/simorgh/blob/latest/src/server/Document/component.jsx#L69-L86 please, so they pull from a single place please? :)

**Review by dr3 — APPROVED** [review]
LGTM, can you update the PR descript to ensure full regression is done on articles & frontpage please :)

File: .storybook/helpers/ampDecorator/index.jsx (hunk 1 above)
Comment (thekp) [reply .storybook/helpers/ampDecorator/index.jsx:21]: https://github.com/bbc/simorgh/issues/2156

File: src/app/containers/ConsentBanner/index.stories.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      platform={platform}
+      isUK
+      origin="https://www.bbc.co.uk"
+      pageType="article"
+      id="c0000000000o"
+      statsDestination="NEWS_PS_TEST"
+      statsPageIdentifier="news.articles.c0000000000o"
+    >
+      <ConsentBanner />
+      This container uses cookies to conditionally render. If you can not see
```
Comment (pjlee11) [inline src/app/containers/ConsentBanner/index.stories.jsx:20]: Nice additional text 😎

Comment (thekp) [reply src/app/containers/ConsentBanner/index.stories.jsx:20]: Props to Drew for the support

**Review by pjlee11 — APPROVED** [review]

**Comment by PriyaKR** [comment]
Looks good to me.

**Pull request merged**